cliquet/tests/test_schema.py

```diff
@@ -51,3 +51,12 @@ def test_ignore_unknwon_fields_by_default(self):
         schema_instance = schema.ResourceSchema()
         deserialized = schema_instance.deserialize({'foo': 'bar'})
         self.assertNotIn('foo', deserialized)
+
+    def test_default_options_parameters_to_ResourceSchema_value(self):
```

Comment:
this should be a sentence. `test_default_options_are_preserved_when_subclassed` ?